ipaserver/plugins/permission.py

```diff
@@ -1238,6 +1238,9 @@ def post_callback(self, ldap, dn, entry, *keys, **options):
         except Exception:
             # Don't revert attribute which doesn't exist in LDAP
             entry.pop('attributelevelrights', None)
+            # NOTE: this relies on the order in relationships to not change
+            for attr in [*self.obj.relationships][2:]:
```

Comment:
May be it is better to have it more explicit:
```
>>> list(filter(lambda a: a not in [1,2], [1,2,3,4,5]))
[3, 4, 5]

```

Reply:
Yes, I knew this would be controversial :-) Thanks for the suggestion, implemented.

I also fixed some linting issues in the test.